internal/testplanet/reconfigure.go

```diff
@@ -37,3 +39,11 @@ var DisablePeerCAWhitelist = Reconfigure{
 		config.Server.UsePeerCAWhitelist = false
 	},
 }
+
+// ShortenOnlineWindow returns a `Reconfigure` that sets the NodeSelection
```

Comment:
after the new test changes I am not sure whether we need this still

Reply:
I would prefer to use this method to have the ability to test the full "grace" period of 1hr per default.
If we set the value to 0 for all tests, we are basically adding the immediate check back, right?